storage/bucket.go

```diff
@@ -827,6 +832,62 @@ type HierarchicalNamespace struct {
 	Enabled bool
 }
 
+// PublicNetworkSource contains the public network source of the
+// bucket's IP filter. Wrapper type for raw.BucketIpFilterPublicNetworkSource.
+type PublicNetworkSource struct {
+	// AllowedIpCidrRanges: The list of public IPv4, IPv6 cidr ranges that are
+	// allowed to access the bucket.
+	AllowedIPCidrRanges []string
+	// ForceSendFields is a list of field names (e.g. "AllowedIpCidrRanges") to
+	// unconditionally include in API requests. By default, fields with empty or
+	// default values are omitted from API requests. See
+	// https://pkg.go.dev/google.golang.org/api#hdr-ForceSendFields for more
+	// details.
+	ForceSendFields []string
+	// NullFields is a list of field names (e.g. "AllowedIpCidrRanges") to include
+	// in API requests with the JSON null value. By default, fields with empty
+	// values are omitted from API requests. See
+	// https://pkg.go.dev/google.golang.org/api#hdr-NullFields for more details.
+	NullFields []string
+}
+
+// VpcNetworkSources contains the VPC network source of the
+// bucket's IP filter. Wrapper type for raw.BucketIpFilterVpcNetworkSources.
+type VpcNetworkSources struct {
+	// AllowedIpCidrRanges: The list of IPv4, IPv6 cidr ranges subnetworks that are
+	// allowed to access the bucket.
+	AllowedIPCidrRanges []string
+	// Network: Name of the network. Format:
+	// projects/{PROJECT_ID}/global/networks/{NETWORK_NAME}
```

Comment:
Are these using a specific project ID or is it a global one like for GCS buckets?